h2/settings.py

```diff
@@ -254,6 +254,13 @@ def __len__(self):
         return len(self._settings)
 
 
+    def __eq__(self,other):
+        return self._settings==other._settings
+
+    def __ne__(self,other):
+        return not self._settings==other._settings
```

Comment:
Actually, you don't want to do that. ;)

The reason is that the rich equality tools in Python have some complex logic, and we don't want to privilege our own execution order. It's better to let Python work it out then to say "you must look at this specific method first".